src/Mod/Arch/ArchComponent.py

```diff
@@ -318,7 +318,7 @@ def getSiblings(self,obj):
 
     def getExtrusionData(self,obj):
 
-        "returns (shape,extrusion vector,placement) or None"
+        "returns (shape,extrusion vector or path,placement) or None"
```

Comment:
Please use triple quotes for the docstring. Since you are modifying it anyway, it should be added. Also, no space after the function declaration.
```py
def getExtrusionData(self,obj):
    """returns (shape,extrusion vector or path,placement) or None"""
```